src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.DesignTime.targets

```diff
@@ -24,9 +24,10 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <!--
       Defines the ability to understand the configuration for the Razor language service provided by
-      the runtime/toolset packages. Introduced in 2.1
+      the runtime/toolset packages. Introduced in 2.1 via a package reference and in 3.0 it's expected
+      to be included by the SDK.
     -->
-    <ProjectCapability Include="DotNetCoreRazorConfiguration"/>
+    <ProjectCapability Include="DotNetCoreRazorConfiguration" Condition="'$(TargetFramework)'=='netcoreapp3.0'"/>
```

Comment:
- I looked but couldn't find any other sensitive areas that got moved over into the SDK. Am I missing any?
- Would love suggestions on how to test this easily, otherwise will just leave untested.

Reply:
Testing this would require putting some MSBuild code in our test projects with a target that cause them to spill their guts via `<Message ...>` we already have good faculties for writing tests that validate output. I would suggest adding something like this - it's very worthwhile.

Reply:
I'd do something like this: https://github.com/aspnet/Razor/blob/master/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Configuration.targets#L47-L49. We don't want to keep maintaining for newer releases

Reply:
There's a couple of tests that do the spilling the guts bit: https://github.com/aspnet/Razor/blob/master/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/BuildIntrospectionTest.cs.

Reply:
Also, it seems as if this file is pulled in for non-design time scenarios. Is that expected?